brave/brave-core
# feat(wallet): Add Account Selector in Send Tab

## Description 
Built out the `Account Selector` for the new `Send` tab.


Resolves <https://github.com/brave/brave-browser/issues/26784>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Go to brave://wallet/send
2. Select an asset to send
3. Click on the `Person` icon in the address bar.
4. Should only show `Accounts` that support the `Selected Asset`
5. Select an available account, the address bar should populate the address.

https://user-images.githubusercontent.com/40611140/202326758-ebda65fd-805d-4fa2-ac96-c867d623595c.mov
